Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DscSignatureValidator.kt

```diff
@@ -60,12 +61,16 @@ class DscSignatureValidator @Inject constructor(
     /**
      * @throws InvalidHealthCertificateException if validation fail, otherwise it is OK!
      */
-    suspend fun validateSignature(dccData: DccData<*>, preFetchedDscData: DscData? = null) {
+    suspend fun validateSignature(
+        dccData: DccData<*>,
+        preFetchedDscData: DscData? = null,
+        date: Date = Date()
```

Comment:
We use Instant or DateTime throughout the app. How come you want to use java Date?